.env

```diff
@@ -0,0 +1 @@
+MONGOHQ_URL=localhost
```

Comment:
How is this file related to integrating Travis CI with the readme badge?

Reply:
I have done necessary changes. Thanks.